### PR TITLE
fix(wallet): adjust max amount to not include network being bridged to

### LINKED ISSUE #20440: Max value calculates 3 networks in bridge flow instead of 2
### Steps:
1. Go to bridge flow
2. Select a network on the 'Bridge to' page to which you are going to bridge assets
3. Go to bridge routes generation
4. Check max value

### Actual result:
The sum of max value consists of 3 networks

### Expected result:
The sum of max value consists of 2 networks, excluding the network selected in 'Bridge to'

image

https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Descoped-Wallet?node-id=15417-73808&t=vW0pAveMDn6J8cax-0

### OS: 
IOS, Android
### Devices:
- Pixel 7a, Android 13
- iPhone 11 Pro Max, IOS 17

### PR DESCRIPTION
**Pull request by J-Son89**
fixes: https://github.com/status-im/status-mobile/issues/20440

This pr adjusts the max amount available for bridging. It only sums up the networks not being sent to. 

Sending should not be affected

Demo of bridging to different networks/ quick show of send flow not being affected
https://github.com/status-im/status-mobile/assets/22799766/28db7c64-c700-4d96-928d-93932477eb86

### CODE REVIEW

File: src/status_im/contexts/wallet/bridge/input_amount/view.cljs [hunk 1]
```diff
@@ -11,14 +11,16 @@
...
-                                        {:amount   amount
-                                         :stack-id :screen/wallet.bridge-input-amount}]))
-     :on-navigate-back  (fn []
-                          (rf/dispatch [:wallet/clean-disabled-from-networks])
-                          (rf/dispatch [:wallet/clean-send-amount])
-                          (rf/dispatch [:navigate-back]))}]])
+    {:current-screen-id      :screen/wallet.bridge-input-amount
+     :button-one-label       (i18n/label :t/review-bridge)
+     :button-one-props       {:icon-left :i/bridge}
+     :enabled-from-chain-ids (rf/sub
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/bridge/input_amount/view.cljs:18]: moved this to a prop of this component - for sending it's configurable and for bridging it's the other networks not being bridged to.

File: src/status_im/subs/wallet/send.cljs [hunk 2]
```diff
@@ -55,3 +55,10 @@
  :wallet/send-token-not-supported-in-receiver-networks?
  :<- [:wallet/wallet-send]
  :-> :token-not-supported-in-receiver-networks?)
+
+(rf/reg-sub
+ :wallet/bridge-from-chain-ids
+ :<- [:wallet/wallet-send]
+ :<- [:wallet/networks-by-mode]
+ (fn [[{:keys [bridge-to-chain-id]} networks]]
+   (filter #(not= % bridge-to-chain-id) (map :chain-id networks))))
```
Comment (ulisesmac) [inline src/status_im/subs/wallet/send.cljs]: nit: by using `keep` we perform a single pass

**Review by ulisesmac — APPROVED** [review]

**Review by BalogunofAfrica — APPROVED** [review]

**Review by OmarBasem — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hi @J-Son89 could you please resolve existing conflicts in the current PR? thanx

**Comment by J-Son89** [comment]
@VolodLytvynenko - done!

**Comment by VolodLytvynenko** [comment]
hi @J-Son89 thank you for PR. The calculation works fine. Could you also exclude the label of network selected on "Bridge <token> to" page  in scope of this PR?

### Steps:
1 . Select network on "Bridge <token> to" page
2. Go to routes generation page of bridge page

### Actual result:
All 3 networks labels are shown
image

### Expected result:
Only two network labels are shown, excluding the network which is selected.
image
https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Descoped-Wallet?node-id=15417-73808&t=vW0pAveMDn6J8cax-0

**Comment by J-Son89** [comment]
Good spot @VolodLytvynenko - I didn't realise that. Should be addressed now 👍

**Comment by VolodLytvynenko** [comment]
hi @J-Son89 Thank you for the fix. The [issue](https://github.com/status-im/status-mobile/pull/20604#issuecomment-2230308184) is resolved. However, one additional issue has been found. Should we be concerned about it? This PR aims to fix the screen where the logic of the max value will be reimplemented as discussed recently.

According to the [new design](https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Descoped-Wallet?node-id=16160-387199&t=c75dFhqYd3ZU9vTU-0), the max value should show only one network, and the current network in the FROM section will not be able to be disabled. Should we just close this PR?

## ISSUE 2: The max value is not updated when network is disabled

### Steps:
1. Go to the routes generation page of the bridge page.
2. Disable a network in the "FROM" section.

### Actual result:
The max value is not updated when a network is disabled.

https://github.com/user-attachments/assets/558909d2-5aaa-46db-a10e-f18688f27905

### Expected result:
The max value should be updated to reflect the available networks after a network is disabled.

### Devices:
Pixel 7a, Android 13
iPhone 11 Pro Max, iOS 17

**Comment by J-Son89** [comment]
@VolodLytvynenko - those designs we went through are for work related to 2.31.
For 2.30 we are keeping the same design to reduce potential bugs and reduce the work overhead so we can focus on bugs.
Cc @churik

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko - those designs we went through are for work related to 2.31. For 2.30 we are keeping the same design to reduce potential bugs and reduce the work overhead so we can focus on bugs. Cc @churik

@J-Son89 Got it. So, [issue 2](https://github.com/status-im/status-mobile/pull/20604#issuecomment-2236223287) is valid and needs to be fixed

**Comment by J-Son89** [comment]
@VolodLytvynenko - with respect to issue 2, what do you think about skipping fixing that for now. The solution to handle this is a bit clunky and we will be removing this by 2.31 as we will only ever have a single network in the max value. Imo it is not worth fixing for the moment.

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko - with respect to issue 2, what do you think about skipping fixing that for now. The solution to handle this is a bit clunky and we will be removing this by 2.31 as we will only ever have a single network in the max value. Imo it is not worth fixing for the moment.

@J-Son89, I agree with you. I thought this issue didn't need a lot of effort. The priority is very low and it can be fixed later. PR can be merged

**Pull request merged**